.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+### Before requesting gatekeeper:
+
+* [ ] Two review approvals and any prior change requests have been resolved.
```

Comment:
There's a GitHub actions hook for "review-requested" so we could check for two +1 approvals when gatekeeper is added and add a comment to the PR saying "please don't do that" which would have the same effect but require zero additional work from gatekeepers.